linear_acceleration_covariance: [0.000002, 0.0, 0.0, 
                                 0.0, 0.000002, 0.0, 
                                 0.0, 0.0, 0.0]
                                 
angular_velocity_covariance: [0.0, 0.0, 0.0, 
                              0.0, 0.0, 0.0, 
                              0.0, 0.0, 0.00005]

pose_covariance: [1, 0.0, 0.0, 0.0, 0.0, 0.0, 
                  0.0, 1, 0.0, 0.0, 0.0, 0.0, 
                  0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 
                  0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 
                  0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 
                  0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
